src/main/java/frc/robot/subsystems/swerve/factories/modules/encoder/RealEncoderConstants.java

```diff
@@ -23,7 +23,7 @@ private static CANcoderConfiguration generateEncoderConfig() {
 		CANcoderConfiguration encoderConfig = new CANcoderConfiguration();
 
 		encoderConfig.MagnetSensor.SensorDirection = SensorDirectionValue.CounterClockwise_Positive;
-		encoderConfig.MagnetSensor.AbsoluteSensorRange = AbsoluteSensorRangeValue.Signed_PlusMinusHalf;
+		encoderConfig.MagnetSensor.AbsoluteSensorDiscontinuityPoint = MathConstants.HALF_CIRCLE.getRotations();
```

Comment:
make sure to test it. just rotate the module for a full rotation and see if it returns values only in the half a circle